package.json

```diff
@@ -36,7 +36,7 @@
     "test": "jest --coverage"
   },
   "peerDependencies": {
-    "react": ">=0.15.0"
+    "react": ">=0.14.0"
```

Comment:
`, <0.15.0` in addition, right?

Reply:
Both ```>=0.14.0, <0.15.0``` ? We only need it to be compatible with 0.14.3 here so I don't understand the need for both. But I might be thinking about this wrong.

Reply:
Oh I think I see now. This is still technically compatible with >0.15.0 though.